src/components/clients/Client.js

```diff
@@ -12,10 +12,38 @@ class Client extends React.Component {
   }
 
   render() {
+    let authRedirectUrls = [];
+    if (this.props.authRedirectUrls) {
```

Comment:
Yes, a client may not have auth redirect URLs since they are optional for the API.